src/commands/game_commands/preset.ts

```diff
@@ -109,7 +134,26 @@ export default class PresetCommand implements BaseCommand {
     }
 
     async loadPreset(presetName: string, guildPreference: GuildPreference, messageContext: MessageContext) {
-        const loadResult = await guildPreference.loadPreset(presetName);
+        let guildID = messageContext.guildID;
+        if (presetName.startsWith("KMQ-")) {
+            // User is loading a preset via UUID
+            const presetUUID = presetName;
+            const existingPresetIdentifiers = await dbContext.kmq("game_option_presets")
+                .select(["guild_id", "preset_name"])
+                .where("option_value", "=", JSON.stringify(presetUUID))
```

Comment:
option_name = uuid?

Reply:
We need to find the preset with the matching uuid, since all presets have a uuid option_name doing that won't work